Components/avatar.css

```diff
@@ -0,0 +1,66 @@
+*{
+    --primary-color:brown;
+}
```

Comment:
Hey, this means that the browser will assign this variable for every block element. instead you can declare this variable in body selector and any element can use it. Rest looks good to me.

Reply:
Thanks @foolhardy21,
done required changes.